src/Bicep.Core.UnitTests/SourceFiles/ResXFileTests.cs

```diff
@@ -33,7 +33,10 @@ private void AssertDoesNotContainTabs(string relativePath, string text)
         [TestMethod]
         public void ResXAndDesignerFilesShouldBeConsistentAndNotCauseUnnecessaryMergeConflicts()
         {
-            string[] resxFiles = System.IO.Directory.GetFiles(BaselineHelper.GetAbsolutePathRelativeToRepoRoot("src"), "*.resx", SearchOption.AllDirectories)
+            // When running in official build, bicep is a Git submodule of the BicepMirror repo.
+            var srcPath = Environment.GetEnvironmentVariable("TF_BUILD") is not null ? "bicep/src" : "src";
+
+            string[] resxFiles = System.IO.Directory.GetFiles(BaselineHelper.GetAbsolutePathRelativeToRepoRoot(srcPath), "*.resx", SearchOption.AllDirectories)
```

Comment:
Good question! It appears that `GetAbsolutePathRelativeToRepoRoot` is used elsewhere for tasks like setting baselines and generating baseline test failure messages. We were fortunate that the issue didn't happen in ADO before because we never updated baselines in ADO, and we didn't trigger official build if there were test failures in GitHub actions. I think it makes more sense to fix `GetRepoRoot` instead.